tox.ini

```diff
@@ -54,15 +55,15 @@ commands = isort --check-only --diff src/django_components
 
 [testenv:coverage]
 deps =
-  pytest-coverage
+  pytest-cov
+  syrupy  # snapshot testing
   # NOTE: Keep playwright in sync with the version in requirements-ci.txt
   playwright==1.48.0
   requests
   types-requests
   whitenoise
 commands =
-  coverage run --branch -m pytest
-  coverage report -m --fail-under=97
+  pytest --cov=django_components --cov-fail-under=87 --cov-branch
```

Comment:
Found this bug with testing coverage. It turned out that we weren't checking coverage of the django-components package, but of the `tests/` directory itself :D

I noticed that because the coverage percentage dropped below 97% with those huge benchmark test files. But as I was looking into it, I saw that the coverage was reporting on the wrong directory. That's why I updated the command and tuned it down to 87%, as the actual coverage is now 89%.